SPECS/dhcp/dhcp.spec

```diff
@@ -1,10 +1,11 @@
 Summary:	    Dynamic host configuration protocol
 Name:		    dhcp
 Version:	    4.4.2
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        MPLv2.0
 Url:      	    https://www.isc.org/dhcp/
 Source0:  	    ftp://ftp.isc.org/isc/dhcp/%{version}/%{name}-%{version}.tar.gz
+Patch1:         CVE-2021-25217.patch
 Group:		    System Environment/Base
```

Comment:
Non-blocking ask: could you lint the spec while you're here? Or at least swap tabs for spaces?